projects/module-3/stretch.md

```diff
@@ -246,151 +246,64 @@ Evals are
+* Git commits are atomic, with concise and precise descriptions of the change made
+* PRs have full, consistent descriptions
+* Team members conduct consistent code reviews of PRs
+* Evolution of the project (decisions made, etc) is documented in the git history and PRs
+* When the project is run locally, the terminal shows no errors and no warnings.
+* The Team collaborates effectively to accomplish the shared goal. The Team productively and professionally works through challenges and conflicts to ensure all team members are able to be heard and contribute throughout the project.
+  * Instructors are available to offer support and guidance but conversations around what is and what is not working are expected to be led by the team members themselves.
+* **Note:** Every team member is expected to fully participate, contribute, communicate and collaborate with the team throughout the entirety of this project. **Failure to do so can result in an individual failing the project, even if the group/project is otherwise passing**.
+
+✨WOW✨ can look like:
+* Mapping out extensions in your project board beyond your MVP
+* Using github issues to track bugs or other issues as they crop up. 
+* Getting several PR reviews from your rocks and/or mentors and implementing their feedback _before_ you merge the PR.
 
 ### React Architecture:
-- [ ] A consistent, modular file structure is used
-- [ ] A clear understanding of class components vs function components is demonstrated
-- [ ] Only the data that a child component _needs_ is passed down as props
-- [ ] Logic is kept out of return statements; return statements are as readable as possible, only communicating what will be displayed
-- [ ] The data displayed on the frontend is consistent with the data stored on the backend
-- [ ] Functions follow the single responsibility principle and are written to be reusable when appropriate
-- [ ] Data fetched from API is cleaned before being set to state
-- [ ] All components which receive props implement prop typechecking (proptypes or otherwise)  
+
+💫ON TRACK💫 can look like:
+* A consistent, modular file structure is used
+* A clear understanding of stateful components vs stateless components is demonstrated
+* Only the data that a child component _needs_ is passed down as props
+* Logic is kept out of return statements; return statements are as readable as possible, only communicating what will be displayed
+* The data displayed on the frontend is consistent with the data stored on the backend
+* Functions follow the single responsibility principle and are written to be reusable when appropriate
+* Data fetched from API is cleaned before being set to state
+* All components which receive props implement prop typechecking (proptypes or otherwise)  
+
+✨WOW✨ can look like:
+* Refactoring a few of your stateful components into class-based components. 
```

Comment:
I don't have an issue with this - but out of curiosity, is the goal to get them experience with class-based components?

Reply:
Yeah, I think we still want our M3 students to know how to use both, since they will likely encounter both on the job. I believe we even have a ticket to create a lesson on class based components somewhere in the pipeline.